ynr/apps/candidates/models/db.py

```diff
@@ -63,6 +56,18 @@ class LoggedAction(models.Model):
         "candidates.Ballot", null=True, on_delete=models.CASCADE
     )
 
+    flagged_type = models.CharField(
+        max_length=100,
+        null=True,
```

Comment:
The django convention is to use `blank=True, null=False` for text fields and use empty string for 'not set' unless there's a genuine reason why `NULL` means one thing and empty string means another. Otherwise you end up with 2 different values denoting 'not set'. Is there a particular case for it here?